### PR TITLE
Fix dappctrl windows build

### PR DESCRIPTION
**Pull request by sofabeat**
Fix statik go generate issues

### CODE REVIEW

File: statik/misc_windows.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+	"github.com/rakyll/statik/fs"
+)
+
+//go:generate .\pkgList\gen_win.bat
+//go:generate go run ../tool/copy_dbscripts/copy.go
+//go:generate cmd /C del /F /Q statik.go
+//go:generate statik -f -src=. -dest=..
+
+// File paths.
```
Comment (ababo) [inline statik/misc_windows.go]: I think we should put the following content into a dedicated file (shared between different architectures).

File: statik/pkgList/gen_win.bat [hunk 2]
```diff
@@ -0,0 +1 @@
+go list ../... > .\pkgList\packages.txt
```
Comment (drew2a) [inline statik/pkgList/gen_win.bat]: If it not a common `statik` file, may be better to move it to the `scripts` folder?

Comment (ababo) [inline statik/pkgList/gen_win.bat]: What is the purpose of this file?

Comment (sofabeat) [reply statik/pkgList/gen_win.bat]: @dzeckelev @ababo - what do you think?

File: statik/misc_windows.go (hunk 1 above)
Comment (sofabeat) [reply statik/misc_windows.go]: ok

**Comment by sofabeat** [comment]
New pull request #320 is opened instead of this one. Closing it.

**Pull request closed without merging**